.github/workflows/ci.yml

```diff
@@ -179,9 +180,28 @@ jobs:
             env: { NO_CYTHON_COMPILE: 1 }
           # Free-threading
           - os: ubuntu-22.04
-            python-version: 3.13-freethreading
-            backend: "c,cpp"
-            env: {}
+            python-version: 3.13t
+            backend: "c"
+            allowed_failure: true
+          - os: ubuntu-22.04
+            python-version: 3.13t
+            backend: "cpp"
+            allowed_failure: true
+          - os: windows-2019
+            python-version: 3.13t
+            backend: "c"
+            allowed_failure: true
+          - os: windows-2019
+            python-version: 3.13t
+            backend: "cpp"
+            allowed_failure: true
```

Comment:
```suggestion
#          - os: windows-2019
#            python-version: 3.13t
#            backend: "c"
#            allowed_failure: true
#          - os: windows-2019
#            python-version: 3.13t
#            backend: "cpp"
#            allowed_failure: true
```

Reply:
Unfortunately this didn't work too well, and just turned of "allowed_failure" I think